qiskit/tools/qcvv/tomography.py

```diff
@@ -694,7 +694,7 @@ def count_keys(n):
 ###############################################################################
 
 
-def fit_tomography_data(tomo_data, method='wizard', options=None):
+def fit_tomography_data(tomo_data, method='wrapper', options=None):
```

Comment:
Seems like the find-and-replace of `wizard` --> `wrapper` picked up some unintended cases here!
This file already had a wizard in it. But it was the only file thankfully. So we should just revert every `wrapper` here to `wizard`.

Reply:
:man_facepalming:  thanks for double-checking the rest of replacement as well!